tmol/kinematics/compiled/compiled.cpu.cpp

```diff
@@ -98,40 +101,40 @@ struct InverseKinDispatch {
     auto dofs = dofs_t.view;
 
     auto k_coords2hts = ([=] EIGEN_DEVICE_FUNC(int i) {
-		if (i==0) {
```

Comment:
i think this (and the failed linting) is due to a change in the version of clang-format. the current env.yml should have the right version.

Reply:
Turns out this was because we've never been linting .cpp files!